tasks/engine_setup.yml

```diff
@@ -24,9 +24,7 @@
     when: ovirt_engine_setup_answer_file_path is defined
 
   - name: Update setup packages
-    package:
-      name: "ovirt*setup*"
-      state: latest
+    command: yum -y update ovirt\*setup\*
```

Comment:
Please add a comment about https://github.com/ansible/ansible/issues/34867 so we can revert do yum module once fixed